File: 05_arrays/05_08_linear_search/linear_search.js
function linearSearch(nums, key) {
    for (let i = 0; i < nums.length; i++) {
        if (nums[i] === key) {
            return i;
        }
    }
    return -1;
}

function print(nums) {
    console.log(nums.join(' '));
}

const nums = [1, 2, 3, 4, 5];
const key = 3;

console.log("Elements:-");
print(nums);
console.log(`The element ${key} is at index: ${linearSearch(nums, key)}`);

File: 05_arrays/05_08_linear_search/linear_search.py
def linear_search(nums, key):
    for i in range(len(nums)):
        if nums[i] == key:
            return i
    return -1

def print_nums(nums):
    print(" ".join(map(str, nums)))

nums = [1, 2, 3, 4, 5]
key = 3

print("Elements:-")
print_nums(nums)
print(f"The element {key} is at index: {linear_search(nums, key)}")
